.github/workflows/haskell.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.4"]
-        cabal: ["3.6.2.0"]
+        ghc: ["8.10.7", "9.2.5"]
+        cabal: ["3.8.1.0"]
```

Comment:
Let's add a `tested-with` field to the `cabal` file, that describes that we've built/tested the package with `ghc-8.10.7` and `ghc-9.2.5` (https://cabal.readthedocs.io/en/3.4/cabal-package.html?highlight=tested-with#pkg-field-tested-with)